inc/cachify.class.php

```diff
@@ -1043,12 +1043,13 @@ private static function _is_index() {
 	 * Check for mobile devices
 	 *
 	 * @since   0.9.1
-	 * @change  2.2.2
+	 * @change  2.3.0
 	 *
 	 * @return  boolean  TRUE if mobile
 	 */
 	private static function _is_mobile() {
-		return ( strpos( TEMPLATEPATH, 'wptouch' ) || strpos( TEMPLATEPATH, 'carrington' ) || strpos( TEMPLATEPATH, 'jetpack' ) || strpos( TEMPLATEPATH, 'handheld' ) );
+		$templatedir = get_template_directory();
```

Comment:
If you set a variable for this, maybe we should unset this var to reduce the storage. However, if users use a storage cache there are faster if you always use the function, not a variable. But I write about so small seconds, so that it is also okay for me in this case.

Reply:
`get_template_director()` has at least 5 levels of nested method calls, 1 option to load and triggers 3 filter hooks, so it **might** perform better. But we can't tell what is actually called and how well OPcache performs here.

Anyway I didn't do any tests here and simply ignored any potential overheads.